### PR TITLE
Adds test for key with trailing slash

### PR DESCRIPTION
**Pull request by JosephSalisbury**

### CODE REVIEW

**Comment by JosephSalisbury** [comment]
@kopiczko i think i'm still missing something here. added test for trailing slash, which passed without changes.

File: storagetest/storagetest.go [hunk 1]
```diff
@@ -9,62 +9,80 @@ import (
...
+			t.Fatal("expected", value, "got", v)
+		}
 
-	v, err = storage.Search(context.TODO(), key)
-	if !microstorage.IsNotFound(err) {
-		t.Fatal("expected", true, "got", false)
-	}
-	if v != "" {
-		t.Fatal("expected", "empty string", "got", v)
+		err = storage.Delete(context.TODO(), key)
```
Comment (kopiczko) [inline storagetest/storagetest.go]: You shouldn't rely on a fact that `Delete` works correctly. Key names should vary between cases.

**Comment by kopiczko** [comment]
I'll test that here: https://github.com/giantswarm/microstorage/pull/5

**Comment by JosephSalisbury** [comment]
@kopiczko is there much point me pushing here, if you're working on https://github.com/giantswarm/microstorage/pull/5?

**Pull request closed without merging**